mobify/progressive-web-scaffold
# Add formatting for phone input on checkout shipping page

This PR adds formatting to the phone number input on the checkout shipping page. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1111
 **Linked PRs**: n/a

## Changes
- Added normalizePhone util 
- Added normalize param to the phone number input

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
-  add an item to your cart
- navigate to: https://www.merlinspotions.com/checkout/
- fill out the form
- there should be formatting when you enter a phone number
- submit the form
- you see the payment page
